draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -76,6 +84,48 @@ public DraggableView(Context context, AttributeSet attrs, int defStyle) {
   }
 
   /**
+   * Return if user can maximize minimized view on click.
+   */
+  public boolean isClickToMaximizeEnabled() {
+    return enableClickToMaximize;
+  }
+
+  /**
+   * Allow to maximize when the user click on minimized view.
```

Comment:
Review this javadoc. This should be something like "Enable or disable click to maximize view when dragged view is minimized".